benchmark/webstreams/readable-async-iterator.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const common = require('../common.js');
+const {
+  ReadableStream,
+} = require('node:stream/web');
+
+const bench = common.createBenchmark(main, {
+  n: [1e5],
+});
+
+
+async function main({ n }) {
+  const rs = new ReadableStream({
+    pull: function(controller) {
+      controller.enqueue(1);
+    }
+  });
+
+  let x = 0;
+  for await (const chunk of rs) {
```

Comment:
Can you move async stuff to a different function like `async-local-storage-run.js`?

Reply:
Why?

Reply:
It is the common method used in several different benchmarks. I assume it was done to keep track of uncaught exceptions easily.